packages/astro/src/@types/astro.ts

```diff
@@ -1,8 +1,8 @@
-import type babel from '@babel/core';
+import type * as babel from '@babel/core';
 import type { z } from 'zod';
 import type { AstroConfigSchema } from '../core/config';
 import type { AstroComponentFactory, Metadata } from '../runtime/server';
-import type vite from '../core/vite';
+import type * as vite from 'vite';
```

Comment:
The idea was to make it easy to switch in & out our own fork of Vite, if we hit a blocking Vite bug. However, I checked out vite.ts and its a pretty simple file to add back in later, so I'm okay with this change. Just calling it out for others.